### ISSUE TITLE
Doubles quote inside image alt are not escaped

### ISSUE THREAD

**Issue by octo-topi** [prismicio/prismic-richtext#47]
### Problem
Using nuxtJs, double quotes inside an image alt are not escaped.
```
<p>
	<img src="<SOME_URL>" alt="Lorem " ipsum" dolor sit amet" >
</p>
```
I expect something similar to
```
<p>
	<img src="<SOME_URL>" alt="Lorem &quot; ipsum &quot; dolor sit amet" >
</p>
```
### Configuration
`@nuxtjs/prismic": "^1.2.4"` 
(package.json in full [here](https://github.com/1024pix/pix-site/blob/dev/package.json))

### Fix
We peeked into the following file, but have not been able to test a fix here
https://github.com/prismicio/prismic-richtext/blob/master/src/astext.ts#L4

**Comment by lihbr** [comment]
Hey @octo-topi, nice catch!

We'll fix that one on the new kits we're working on. In the meantime, I suggest you fixing that on your [HTML serializer](https://prismic.io/docs/technologies/html-serializer-nuxtjs)

Let us know if anything!

**Comment by lihbr** [comment]
Hey, a patch has been released: [`prismic-dom@2.2.6`](https://github.com/prismicio/prismic-dom/releases/tag/v2.2.6) addressing this issue.

Running `npm update` / `yarn upgrade` should get you the latest version of it (alternatively you can delete your lock file and install dependencies again)

This has also been addressed here on our new kits (currently in alpha): https://github.com/prismicio/prismic-helpers/pull/13/commits/c0e6d6d6d8d36747de381cc6acd3c7fec6d839d2
